unit_tests/common/test_tuple_util.cpp

```diff
@@ -214,6 +214,14 @@ namespace gridtools {
             EXPECT_EQ(90, acc);
         }
 
+        TEST(concat, functional) {
+            EXPECT_EQ(concat(std::make_tuple(1, 2), std::make_tuple(3, 4)), std::make_tuple(1, 2, 3, 4));
```

Comment:
Is that enough, only one test? Thinking about corner cases (empty tuple or so).

Reply:
Added one more. Ready to add more on request.